python/cn_tn.py

```diff
@@ -501,6 +501,26 @@ def fraction2chntext(self):
         numerator, denominator = self.fraction.split('/')
         return num2chn(denominator) + '分之' + num2chn(numerator)
 
+class Time:
+    """
+    Time类
+    """
+    def __init__(self, time=None, chntext=None):
+        self.time = time
+        self.chntext = chntext
+    def chntext2time(self):
+        hours, minutes = self.chntext.split('点')
+        return chn2num(hours) + ':' + chn2num(minutes)
+    def time2chntext(self):
+        hours, minutes = self.time.split(':')
+        if len(hours) == 2 and hours[0] == '0':
+            hours = hours[1]
+        if int(minutes) < 11 and int(minutes) > 0:
```

Comment:
can you please give me an example what is this 11 for? and also the residual operator to 11.